src/ploigos_step_runner/step_implementers/shared/openscap_generic.py

```diff
@@ -107,21 +111,21 @@ class OpenSCAPGeneric(StepImplementer):
     #    Result	fail
     #
     # Matches:
-    #    (Title	RHSA-2020:4186: spice and spice-gtk security update (Important)
+    #    (Title	RHSA-2020:4186: spice and spice-gtk security update ((Important))
     #    Rule	xccdf_com.redhat.rhsa_rule_oval-com.redhat.rhsa-def-20204186
     #    Ident	RHSA-2020:4186
     #    Ident	CVE-2020-14355
     #    Result	(pass))
     #
-    #    (Title	RHSA-2020:3658: librepo security update (Important)
+    #    (Title	RHSA-2020:3658: librepo security update ((Important))
```

Comment:
why the extra set of parenthases added here?

Reply:
The text from the results have a parenthesize and parenthesize were also used in the comment to show were the reqexp groupings are so this ends up with doubles. So we'd either need to change the group delimiter or not show the group to get around the double.

Reply:
doh, ignre me.